os.path/os/path.py

```diff
@@ -79,6 +80,13 @@ def islink(path):
     except OSError:
         return False
 
+def realpath(path):
+    libc = ffilib.libc()
+    if isinstance(path, str) or isinstance(path, bytes):
+        realpath_ = libc.func("s", "realpath", "ss")
+        #NOTE: memory leak! should free() returned pointer, see man realpath
+        return realpath_(path, None)
```

Comment:
Btw, note that you took care to recognize both `str` and `bytes` (and only them, e.g. `bytearray` won't work), but not taking care to return bytes-typed result for bytes arg.

That's ok for "first version", I'm merging it as is, further issues can be addressed as actually hit in practice ;-).